### ISSUE TITLE
Display the "Save" button at all times allowing users to select only after saving any data in the wizard

### ISSUE THREAD

**Issue by VJ1285** [hackforla/tdm-calculator#777]
### Overview

Display the "Save" button all times, allowing users to select only after saving any data in the wizard. Currently The save button is only displayed after parking is entered, but it should be always visible and greyed out if no changes have been made since previous a save.

### Action Items

- [x] Create Greyed out Save button icon in the same style as greyed out arrow buttons 
- [x] Set up method for determining when anything has been changed on the project
- [x] Display Yellow Save button when users can save a project. 

### Resources/Instructions

Current Yellow Save button:
![Screen Shot 2021-08-01 at 3 33 02 PM](https://user-images.githubusercontent.com/49739276/127787305-8d6183a2-4865-4db4-9364-ac0c8daf2519.png)

Greyed out arrow button:
![Screen Shot 2021-08-01 at 3 32 56 PM](https://user-images.githubusercontent.com/49739276/127787306-ca33b0a6-084f-43b8-8f8d-2ab77c6e0a32.png)

**Comment by seenaiype** [comment]
@fyliu , I was checking this on dev, for guest users, the "Save project" is not doing anything and I am guess a person would lose all the data if he tries to create an account in the middle. Since we already have a a warning on the top, we feel this might be redundant. Can you make the below update, let me know if any questions or concerns. 
 
- [ ] remove the "Save Project" for the guest users

Moving back to in-Progress

**Comment by fyliu** [comment]
@seenaiype Sounds good. That's right. The save button would show up disabled for guest users or users that forgot to login. You're right that creating an account or logging in at this point would cause data loss, which is not ideal.

I added a new pull request with a screenshot of the change.

It's kind of weird now since this issue is called "display the save button at all times..." :shrug:

**Comment by seenaiype** [comment]
I think, the issue was the save project was not being shown on all pages , for logged in users.

Let's remove the button for u
Not logged in users, as it will cause confusion with no tool tip etc..

**Comment by fyliu** [comment]
Thanks. It's ready for visual review. Screenshot in the new pull request. It's hidden on all pages for guests.

**Comment by ExperimentsInHonesty** [comment]
Demoed to Stakholder (Dave/Alexander)
